animation_nodes/nodes/color/c_utils.pyx

```diff
@@ -0,0 +1,27 @@
+# cython: profile=True
+cimport cython
+
+from ... data_structures cimport (
+    ColorList,
+    VirtualColorList,
+    PolygonIndicesList
+)
+
+def loopsColorsList(long Amount, PolygonIndicesList polygons, VirtualColorList colors, 
```

Comment:
Use a more descriptive name, `getLoopColorsFromVertexColors`. The same for the other function.

Reply:
I have renamed both of the functions.